ocs_ci/ocs/scale_lib.py

```diff
@@ -267,7 +268,11 @@ def create_scale_pods(
         self.ms_name = list()
 
         # Check for expected worker count
-        expected_worker_count = get_expected_worker_count(scale_count)
+        if is_managed_service_cluster():
+            expected_worker_count = 3
+        else:
+            expected_worker_count = get_expected_worker_count(scale_count)
+
```

Comment:
This change looks ok, would like to know if we will create more than 1500+ PVCs and more than 76+ pods.

Reply:
Currently, we don't need to use more than 1500 PVCs.